db/migrate/20201006084522_remove_setting_id_index_from_delegations.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveSettingIdIndexFromDelegations < ActiveRecord::Migration[5.2]
+  def change
+    remove_index :accounts, name: "index_decidim_delegations_on_action_delegator_setting_id"
```

Comment:
:warning: wrong table. You meant `decidim_action_delegator_delegations`, right?

Reply:
🤦